rclcpp/test/test_node.cpp

```diff
@@ -25,13 +25,23 @@
 #include "rclcpp/scope_exit.hpp"
 #include "rclcpp/rclcpp.hpp"
 
+#include "rcpputils/filesystem_helper.hpp"
+
```

Comment:
nit: delete extra new line

Reply:
Done in d5ac56e7e6c2548f76386dc71516bc4d48cf7487